modular_skyrat/modules/loadouts/loadout_items/loadout_datum_heads.dm

```diff
@@ -558,6 +558,11 @@ GLOBAL_LIST_INIT(loadout_helmets, generate_loadout_items(/datum/loadout_item/hea
 	item_path = /obj/item/clothing/head/sec/peacekeeper/sol
 	restricted_roles = list(JOB_SECURITY_OFFICER, JOB_SECURITY_MEDIC, JOB_HEAD_OF_SECURITY, JOB_WARDEN)
 
+/datum/loadout_item/head/nriofficercap
+	name = "Security Officer's NRI Cap"
+	item_path = /obj/item/clothing/head/beret/sec/nri
+	restricted_roles = list(JOB_SECURITY_OFFICER, JOB_SECURITY_MEDIC, JOB_HEAD_OF_SECURITY, JOB_WARDEN)
+
```

Comment:
Security is hired by Lopland and armored by Amradyne, both jobs that operate within SolFed space. Why would they ever think to allow their staff to wear anti-SolFed uniforms, in SolFed space.?

Reply:
Oh right I forgot.

Reply:
Doesn't exist anymore tbh.